pkg/compose/build.go

```diff
@@ -81,6 +81,14 @@ func (s *composeService) build(ctx context.Context, project *types.Project, opti
 				Attrs: map[string]string{"ref": image},
 			})
 		}
+		if len(buildOptions.Platforms) > 1 {
+			buildOptions.Exports = []bclient.ExportEntry{{
+				Type: "image",
+				Attrs: map[string]string{
+					"push": "true",
```

Comment:
Does this mean that images are automatically pushed during a `docker compose build`?

Reply:
Good catch, I added that to check e2e tests but as the default behaviour of Compose is to not push by default I will remove that and update the tests
Thanks @nicksieger